board.rb

```diff
@@ -45,6 +47,39 @@ def find_person
   attr_reader :cells
 
   def available?(cell)
-    cells[cell] == OPEN
+    cells[cell] == OPEN || cells[cell] == STORAGE
+  end
+
+  def has_crate?(cell)
+    cells[cell] == CRATE || cells[cell] == CRATE_ON_STORAGE
+  end
+
+  def move_person(person, index)
+    update_previous_location(person, index)
+    move_to_new_location(person, index)
+  end
+
+  def move_crate(person, index)
+    if cells[find_person + index] == OPEN
+      cells[find_person + index] = CRATE
+    else
+      cells[find_person + index] = CRATE_ON_STORAGE
+    end
+  end
+
+  def update_previous_location(person, index)
```

Comment:
Unused method argument - `index`. If it's necessary, use `_` or `_index` as an argument name to indicate that it won't be used.